build.gradle.kts

```diff
@@ -151,7 +151,7 @@ tasks.dokkaHtmlMultiModule {
 
 publishing {
   publications {
-    create<MavenPublication>("tbdex") {
+    create<MavenPublication>(rootProject.name) {
       groupId = project.group.toString()
       artifactId = name
       description = "Kotlin SDK for tbdex functionality"
```

Comment:
thinking i should somehow enable us to package all 3 projects exported under 1 name - `tbdex`. looking into shadow jars

Reply:
nvm found how to do it [here](https://github.com/TBD54566975/tbdex-kt/pull/152/files#r1483160314)